website/static/admin/config.yml

```diff
@@ -93,7 +93,7 @@ collections:
       - { label: "Name of Organization", name: "organization", widget: "string" }
       - { label: "Location", name: "location", widget: "string" }
       - { label: "Show post", name: "show", widget: "boolean", default: true }
-      - { label: "Description", name: "description", widget: "markdown", hint: "Please describe the opportunity" }
+      - { label: "Description", name: "body", widget: "markdown", hint: "Please describe the opportunity" }
```

Comment:
The CMS wasn't writing this field correctly to the markdown file - hoping this solves that.